examples/LoRaWAN/LoRaWAN_ABP/configABP.h

```diff
@@ -64,32 +64,41 @@ const uint8_t subBand = 0;  // For US915, change this to 2, otherwise leave on 0
   #pragma error ("ARDUINO_TBEAM_USE_RADIO_SX1262 awaiting pin map")
 
 #elif defined(ARDUINO_TBEAM_USE_RADIO_SX1276)
-  #pragma message ("Using TTGO LoRa32 v2.1 marked T3_V1.6.1 + Display")
+  #pragma message ("Using TTGO T-Beam")
   SX1276 radio = new Module(18, 26, 23, 33);
 
 
-// Heltec
+// HelTec: https://github.com/espressif/arduino-esp32/blob/master/variants/heltec_*/pins_arduino.h
 #elif defined(ARDUINO_HELTEC_WIFI_LORA_32)
-  #pragma error ("ARDUINO_HELTEC_WIFI_LORA_32 awaiting pin map")
-
-#elif defined (ARDUINO_heltec_wireless_stick)
-  #pragma message ("Using Heltec Wireless Stick")
-  SX1278 radio = new Module(14, 4, 12, 16);
+  #pragma message ("Using Heltec WiFi LoRa32")
+  SX1276 radio = new Module(18, 26, 14, 33);
 
-#elif defined(ARDUINO_heltec_wifi_lora_32_V2)
+#elif defined(ARDUINO_HELTEC_WIFI_LORA_32_V2)
   #pragma message ("Using Heltec WiFi LoRa32 v2")
-  SX1278 radio = new Module(14, 4, 12, 16);
```

Comment:
This was contributed in #1047 and according to @matthias-bs it was tested. So which pin map (and module type)¨is correct then?

Reply:
Oh! From @matthias-bs! I know his Bresser projects which are really cool!

I just tested this with my HelTec LoRa32v2: Using pin name definitions (from arduino-esp32) and plain numbers work:
`SX1276 radio = new Module(18, 26, 14, 35);`
`SX1276 radio = new Module(SS, DIO0, RST_LoRa, DIO1);`

Transmit/Receive (FSK) works. As do the interrupts via `fifoGet`.

When looking at the [Bresser board definitions](https://github.com/matthias-bs/BresserWeatherSensorReceiver/blob/main/src/WeatherSensorCfg.h#L454) you can see that he is also using the _arduino-esp32_ names instead of numbers. I can't see any error with my PIN definitions?

When compiling (ArduinoIDE 2.x) the board flag gets set in all caps: `-DARDUINO_HELTEC_WIFI_LORA_32_V2`

Regarding module type: Most LoRa32 boards use the SX1276 cause of the wider freq range.
But when poking the version register it identifies as a SX1278. I can cross check with a _STM32L151CCU6_ board (_HelTec LoRa 151_) if you want?

PS: Thanks for this wonderful lib! Great work!

Reply:
To me this is still not resolved. @matthias-bs can you comment on this? Is your set of pins correct, or is @Velocet's? It seems to me like it should be the same ahrdware, so I don't understand why there are two different sets of pins.

If there is some discrepancy then I will have to err on the side of caution and just remove the pin maps.

Reply:
Based on [pinmap] (https://resource.heltec.cn/download/WiFi_LoRa_32/WIFI_LoRa_32_V2.pdf) and [schematic](https://resource.heltec.cn/download/WiFi_LoRa_32/V2/WIFI_LoRa_32_V2(868-915).PDF) I'd say the SX1276 is correct. And it is an SX1276 as can be seen from the schematic. But I don't have one, so can't test for 100% certainty.

Reply:
> If there is some discrepancy then I will have to err on the side of caution and just remove the pin maps.

I've got all the files open on the sofa laptop ready for some wine based PR's tonight, including correcting a couple of my own errors. Should I compile before pushing?

Reply:
> To me this is still not resolved. @matthias-bs can you comment on this? Is your set of pins correct, or is @Velocet's? It seems to me like it should be the same ahrdware, so I don't understand why there are two different sets of pins.

I tested this with my HelTec WiFi LoRa32 V2 and other projects use the same PINs as do other people who use RadioLib:
- (SX1276 unable to receive packets)[https://github.com/jgromes/RadioLib/discussions/371]
- [Problem with 'receiveDirect()' on SX1276](https://github.com/jgromes/RadioLib/issues/276)

The values are the same for the [HelTec](https://github.com/Heltec-Aaron-Lee/WiFi_Kit_series/tree/master/variants) and [arduino-esp32](https://github.com/espressif/arduino-esp32/tree/master/variants) PIN definitions. One thing to mention: DIO1 and DIO2 on the HelTec are connected to Input only PINs which could result in errors handling interrupts.

> If there is some discrepancy then I will have to err on the side of caution and just remove the pin maps.

The PIN maps are a great value for beginners and should definitely stay. I also see no discrepancy: Use the "official" PIN definitions from arduino-esp32 and you are good to go without any headaches.

PS: only have my mobile with me. Can't do the readme commit deletion before sunday 🫠

Reply:
> To me this is still not resolved. @matthias-bs can you comment on this? Is your set of pins correct, or is @Velocet's? It seems to me like it should be the same ahrdware, so I don't understand why there are two different sets of pins.
> 
> If there is some discrepancy then I will have to err on the side of caution and just remove the pin maps.

I could not test any Heltec board myself - I don't have any and I won't by one anytime soon. The Heltec definitions are based on documentation and feedback by users of my project.

Reply:
I have a V2, which is where the original pins came from so I'm very sure that it does work but as I selected the wifi_kit rather than wifi_lora when originally fishing out the defines. The wifi_lora has been added, I'm not sure where those pins have come from, possibly from the wireless stick.